events/api.py

```diff
@@ -904,22 +904,30 @@ class OrganizationSerializer(LinkedEventsSerializer):
     )
     sub_organizations = serializers.HyperlinkedRelatedField(
         queryset=Organization.objects.all(),
-        source='children',
         view_name='organization-detail',
         many=True,
     )
+    affiliated_organizations = serializers.HyperlinkedRelatedField(
+        queryset=Organization.objects.all(),
+        view_name='organization-detail',
+        many=True,
+    )
+    is_affiliated = serializers.SerializerMethodField()
 
     class Meta:
         model = Organization
         fields = (
             'id', 'data_source', 'origin_id',
             'classification', 'name', 'founding_date',
             'dissolution_date', 'parent_organization',
-            'sub_organizations', 'responsible_organization',
+            'sub_organizations', 'affiliated_organizations',
```

Comment:
I believe we should test these two fields, `sub_organizations` and `affiliated_organizations`, in the `TestOrganizationSerializer`. Currently, both fields have the same queryset so I wasn't sure that `sub_organizations` only lists proper suborganizations and `affiliated_organizations` only lists affiliated ones.